lang/en/responses.php

```diff
@@ -14,6 +14,8 @@
     'deletion_successful' => 'Successfully deleted media.',
     'image_upload_successful' => 'Successfully uploaded new image version.',
     'image_version_set' => 'Successfully set image version.',
+    'pdf_upload_successful' => 'Successfully uploaded new PDF version.',
```

Comment:
maybe turn this into the document nomenclature

Reply:
done